src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+import core.basesyntax.abstractions.Figure;
+import core.basesyntax.suppliers.FigureSupplier;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class Main {
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[6];
+        FigureSupplier figureSupplier = new FigureSupplier();
+        for (int i = 0; i < figures.length; i++) {
+            if (i < figures.length / 2) {
+                figures[i] = figureSupplier.getRandomFigure();
+            } else {
+                figures[i] = figureSupplier.getDefaultFigure();
+            }
+        }
+        for (Figure figure : figures) {
```

Comment:
no need to create extra loop
```suggestion
            }
```